test/app.test.js

```diff
@@ -372,10 +410,51 @@ describe("app", function() {
             it('should accept valid options', function () {
                 return tester
                     .setup.user.state('No_SA_ID_Gender_Entry')
+                    .setup.user.answers(completed_answers)
                     .input('1')
                     .check.reply.content(/Thank you! Your report has been submitted/)
                     .run();
             });
         });
+
+        describe('Submit_Case', function () {
+            it('should make a valid Ona API call', function () {
+                return tester
+                    .setup.user.state('Submit_Case')
+                    .setup.user.answers(completed_answers)
```

Comment:
Loading all the "answers" the user would've given in earlier states so we have a complete dictionary of stuff we'd be posting to Ona.